client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/style.scss

```diff
@@ -112,11 +112,17 @@
 }
 
 .packages-step__package-weight {
-	display: inline-block;
 	width: 140px;
-	margin-bottom: 0;
+	margin-bottom: 10px;
+	float: left;
+	margin-right: 65px;
 }
 
 .packages-step__package-weight-unit {
 	margin-left: 8px;
 }
+
+.packages-step__package-signature {
+	width: 190px;
```

Comment:
On smaller screens, the width of this block and the weight block don't align:

<img width="643" alt="weight-and-signature-blocks-dont-align-screenshot" src="https://user-images.githubusercontent.com/11487924/37726529-fd4ecda0-2d0b-11e8-9f6a-215c7b7352b2.png">


Should we make it 140px instead to align?

Reply:
Oh interesting.. this is browser-dependent.

Firefox:

<img width="217" alt="screen shot 2018-03-23 at 1 56 48 pm" src="https://user-images.githubusercontent.com/63922/37850779-1abfe896-2ea2-11e8-87f3-827c3a74e719.png">

Safari:

<img width="212" alt="screen shot 2018-03-23 at 1 57 28 pm" src="https://user-images.githubusercontent.com/63922/37850804-312f2bb4-2ea2-11e8-91de-a986e0877c6c.png">

Chrome:

<img width="217" alt="screen shot 2018-03-23 at 1 58 04 pm" src="https://user-images.githubusercontent.com/63922/37850819-3ba5c256-2ea2-11e8-8e30-3479ff106cd3.png">

Reply:
Discussed this with @kellychoffman - we'll fix for Safari and Chrome and let the width be off in Firefox.

Reply:
Cool. It still doesn't work for me in FF, but it's a small issue that we were happy to merge with anyway, so it's all good.